src/constants.cairo

```diff
@@ -0,0 +1,57 @@
+// Config ID to fetch world config
+const WORLD_CONFIG_ID: felt252 = 999999999999999999;
+const BUILDING_CONFIG_ID: felt252 = 999999999999999998;
+const LABOR_CONFIG_ID: felt252 = 999999999999999997;
+
+// 8 bits
+const RESOURCE_IDS_PACKED_SIZE: usize = 8_usize;
+const REALMS_DATA_PACKED_SIZE: usize = 8_usize;
+
+// prime 
+const PRIME: felt252 = 3618502788666131213697322783095070105623107215331596699973092056135872020480;
```

Comment:
To be fair, this is max(felt252), not the Cairo prime. It doesn't seem to be used anywhere either.

Reply:
my bad, forgot to remove. is prime = 2**251 + (17*2**192) + 1 still true ?

Reply:
Yes, still is.